app/experimenter/nimbus-ui/src/components/App/index.tsx

```diff
@@ -3,22 +3,17 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
-import AppLayout from "../AppLayout";
 import { Router } from "@reach/router";
 
 import PageHome from "../PageHome";
-import PageExperimentNew from "../PageExperimentNew";
+import PageNew from "../PageNew";
 
 const App = ({ basepath }: { basepath: string }) => {
   return (
-    <div data-testid="app">
-      <AppLayout>
```

Comment:
Realized that since we have `AppLayout` vs `AppLayoutExperimentEdit`, each `Page*` component will want to select its own layout.